Packs/qualys/Integrations/Qualysv2/Qualysv2.py

```diff
@@ -1684,6 +1684,8 @@ def get_host_list_detection(self, since_datetime, next_page=None, limit=HOST_LIM
         params: dict[str, Any] = {
             "truncation_limit": limit,
             "vm_scan_date_after": since_datetime,
+            "show_qds": 1,
+            "show_qds_factors": 1,
```

Comment:
why is it hard coded value?

Reply:
Based on what we have discussed in person, I have added a comment to explain these two params